pegasus/test/test_section_api_helpers.rb

```diff
@@ -234,6 +234,94 @@ class SectionApiHelperTest < SequelTestCase
   end
 
   describe DashboardUserScript do
-    # TODO(asher): Add tests here.
+    before do
+      FakeDashboard.use_fake_database
+      @script_id = 1
```

Comment:
does this value have particular meaning, or did we just need some arbitrary script_id?